1/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace _1
+{
+    class Program
+    {
+        public class Program
+        {
+            public static void Main(string[] args)
+            {
+
+                double x = 0;
+                double pi = 3.14159265358979323846264;
```

Comment:
Используйте константу из класса `Math`
```suggestion
                double pi = Math.PI;
```